### ISSUE TITLE
what you need for this to work

### ISSUE THREAD

**Issue by McJonnycat** [LEVIXDDLMAO/FNF-FunkedEngine#2]
you will need 
1. the files  for the source

**Comment by LEVIXDDLMAO** [comment]
yeah i know i been tested code

**Comment by LEVIXDDLMAO** [comment]
Yo so i wanna make a charater changer for the engine and try the mod folder for html

**Comment by McJonnycat** [comment]
first of all

**Comment by McJonnycat** [comment]
the mod folder is used using filesystem

**Comment by McJonnycat** [comment]
so  ye

**Comment by McJonnycat** [comment]
and also how am i gonna acces this

**Comment by McJonnycat** [comment]
if your just telling me this

**Comment by McJonnycat** [comment]
if your doing the character changer on lua then you whould need to use this linc_luajit for html5 https://github.com/AndreiRudenko/linc_luajit

**Comment by LEVIXDDLMAO** [comment]
okay that would work we just need to add options

**Comment by LEVIXDDLMAO** [comment]
i seen the push

**Comment by McJonnycat** [comment]
oh ok

**Comment by McJonnycat** [comment]
also you need the assets

**Comment by McJonnycat** [comment]
and plus just get the github app

**Comment by McJonnycat** [comment]
so you can upload the files

**Comment by McJonnycat** [comment]
https://desktop.github.com/

**Comment by LEVIXDDLMAO** [comment]
yeah im working on the source first to make it easyer sorry i cant spell for my life

**Comment by McJonnycat** [comment]
its ok

**Comment by LEVIXDDLMAO** [comment]
Dont forget to add your self to the read me

**Comment by McJonnycat** [comment]
ok

**Comment by LEVIXDDLMAO** [comment]
your apart of the team

**Comment by McJonnycat** [comment]
yeeeee

**Comment by LEVIXDDLMAO** [comment]
lmao

**Comment by SanicBTW** [comment]
i saw some documentation about filesystem access on html5 and it requires "hxnodejs" or something like that but i havent tested it

**Comment by SanicBTW** [comment]
but maybe instead of adding the mods folder we could put the mods folder inside the assets folder but i dont know if it'll work after compilation

**Comment by LEVIXDDLMAO** [comment]
yeah that might work

**Comment by LEVIXDDLMAO** [comment]
SanicBTW add your self to the read me

**Comment by LEVIXDDLMAO** [comment]
and for a early version for the game i want milf to be the only song playable for a little

**Comment by Dominicthecodingenuis** [comment]
ok

**Comment by LEVIXDDLMAO** [comment]
Yeah then we can work on a mod for the engine

**Comment by SanicBTW** [comment]
> SanicBTW add your self to the read me

Will add myself once I get home, I'm outside rn

**Comment by LEVIXDDLMAO** [comment]
okat

**Comment by SanicBTW** [comment]
Yo Levi if you want I could upload some engine source code and be able to modify it

**Comment by LEVIXDDLMAO** [comment]
Yeah thats fine

**Comment by LEVIXDDLMAO** [comment]
and plus dont forget to add your self to read me criedts

**Comment by LEVIXDDLMAO** [comment]
heres the todo list
- make a early version of the engine with milf as the only song
- special notes 
- charater changer 
- adding the flies
- custom note colors like psych engine
- adding memory counter
- higher fps for html5 
- being able to optomise the engine for low end pc 
- adding andorid support
- make the full engine
- add week 8 
- maybe add a mod folder to html
- song time
- a few mod weeks 
- then a mod for the engine 
- easy chart editor
- Chanellges like FNF ONLINE VS
- finshing the engine and doing a full relese of the engine
- if bugs are found fixing them 
- adding things that the ppl want 
thats really all its long for a good reason i want this to blow up and be really good

**Comment by LEVIXDDLMAO** [comment]
Dont worry abour the ranbo0 person thats me

**Comment by Dominicthecodingenuis** [comment]
is ranbo0 your alt

**Comment by LEVIXDDLMAO** [comment]
kinda i use replit to upload my shit because its super easy its like github desktop

**Comment by LEVIXDDLMAO** [comment]
what did you add to the Project.xml? im just wondering

**Comment by Dominicthecodingenuis** [comment]
the fps

**Comment by LEVIXDDLMAO** [comment]
oh okay i was just wondering, how much fps would html get?

**Comment by LEVIXDDLMAO** [comment]
hey team in the wiki i added the to do list if that helps

**Comment by LEVIXDDLMAO** [comment]
and whos wants to finsh the read me?? and sanicBTW you can add stuff

**Comment by LEVIXDDLMAO** [comment]
how would we add options??

**Comment by Dominicthecodingenuis** [comment]
idk

**Comment by LEVIXDDLMAO** [comment]
yeah i add some more files like the change log

**Comment by Dominicthecodingenuis** [comment]
question is intro.txt in assets or mods

**Comment by LEVIXDDLMAO** [comment]
ask @SanicBTW  I dont really know i been just adding things

**Comment by LEVIXDDLMAO** [comment]
uploading preload for you dom so you can edit the intro text

**Comment by LEVIXDDLMAO** [comment]
how would i take away the weeks for the early version?

**Comment by LEVIXDDLMAO** [comment]
yall good?

**Comment by LEVIXDDLMAO** [comment]
Dom So i uploaded the intro.txt for ya its in assets/preload @Dominicthecodingenuis

**Comment by LEVIXDDLMAO** [comment]
Do i just lock the other weeks? and we need note splashes

**Comment by LEVIXDDLMAO** [comment]
as i said i want milf to be the only one playable for the early version of the engine

**Comment by Dominicthecodingenuis** [comment]
ok i will make milf the only song

**Comment by LEVIXDDLMAO** [comment]
okay i got to re add source i lost track

**Comment by Dominicthecodingenuis** [comment]
ok

**Comment by LEVIXDDLMAO** [comment]
just finshing up assets

**Comment by McJonnycat** [comment]
we should leave this for the discussion area

**Comment by LEVIXDDLMAO** [comment]
yeah

**Comment by McJonnycat** [comment]
ok i made a dev talk in the discussions area

**Comment by LEVIXDDLMAO** [comment]
yeah i seen it team go to discussions

**Comment by SanicBTW** [comment]
> ask @SanicBTW  I dont really know i been just adding things 
> 

Bro I fell asleep
